Maquetado/views/login.html

```diff
@@ -0,0 +1,35 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <title>Login</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet" type="text/css" href="../application.css">
+</head>
+
+<body>
+  <div class='pageContainer'>
+    <div class='loginContainer'>
+      <img src='../resources/img/LogoWolox.png' class='woloxLoginLogo' alt='Wolox Logo Image' />
+      <div class='inputsContainer'>
+        <h2 class='dataTitle'>Email</h2>
+        <input class='dataInput'>
+        <h2 class='dataTitle'>Password</h2>
+        <input class='dataInput'>
+        <div class='loginButton'>
+          <a href='http://127.0.0.1:5500/Maquetado/views/bookList.html'>
+            <h3 class='loginTitle'>Login</h3>
```

Comment:
There's no need to put a h3 inside the anchor (it may be worth it if you habe a heading with a link, but in this case in my opinion the button texts should not be a heading). Also the properties applied by "loginTitle" can be applied to the anchor too (you'll need to modify "loginButton" if you choose to apply it as the class of `<a>`).

```html
<a class="loginButton" href='http://127.0.0.1:5500/Maquetado/views/bookList.html'>
  Login
</a>
```